src/main/scala/UniqueAddition.scala

```diff
@@ -0,0 +1,28 @@
+object UniqueAddition extends App{
+  var sum = 0
+
+  def add(a:Int, b:Int, c:Int):Int={
```

Comment:
a bit too much of ifs, try to see if you can get it to rework to be a patternmatching solution

a, b, c are not good names